guppylang/std/qsystem/__init__.py

```diff
@@ -25,30 +38,72 @@ def phased_x(q: qubit, angle1: angle, angle2: angle) -> None:
 @guppy
 @no_type_check
 def zz_max(q1: qubit, q2: qubit) -> None:
-    """Maximally entangling ZZPhase.
-
-    Equivalent to `zz_phase(q1, q2, pi / 2)`.
+    r"""zz_max gate command. A maximally entangling zz_phase gate.
+
+    This is a special case of the ZZPhase operation with angle = pi/2
```

Comment:
Let's be consistent about capitalization in the op names (here we have both `zz_phase` and `ZZPhase`).

Reply:
Not addressed.